### PR TITLE
Fix S1 test default rule verificaton

### PR DESCRIPTION
**Pull request by koolzz**
Signed-off-by: Nick Yurchenko <koolzz@fb.com>

## Summary

After I increased the ipfix flow priority the test fails, reducing the tbl0 priority0 flow count verification to 1

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

File: lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py [hunk 1]
```diff
@@ -586,7 +586,7 @@ def verify_flow_rules_deletion(self):
             dpath, {"table_id": self.SPGW_TABLE, "priority": 0}
```
Comment (koolzz) [inline lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py]: this`verify_flow_rules_deletion` function seems odd, I'm assuming we're verifying that all rules were removed? Should we remove the priority match here completely then?

Or we could potentially query rules of different priorities (assert that no UE rules are left)
@ulaskozat

Comment (koolzz) [reply lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py]: Lets merge this PR to unblock CI though

Comment (ulaskozat) [reply lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py]: let's merge it and can discuss after unblocked for the best way forward.

**Review by ulaskozat — APPROVED** [review]
LGTM

File: lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py (hunk 1 above)
Comment (ulaskozat) [reply lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py]: but yes, why do we check against priority 0? @pruthvihebbani

Comment (ulaskozat) [reply lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py]: @koolzz on table zero no rules but the default rule(s) should remain. removing "priority":0 in the match criteria should fix the logic.

Comment (koolzz) [reply lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py]: sure nuked the priority match lets see

File: lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py [hunk 2]
```diff
@@ -583,7 +583,7 @@ def verify_flow_rules_deletion(self):
         print("Checking if all uplink/downlink flows were deleted")
         dpath = get_datapath()
         flows = get_flows(
-            dpath, {"table_id": self.SPGW_TABLE, "priority": 0}
+            dpath, {"table_id": self.SPGW_TABLE}
         )
         assert(
             len(flows) == 2), "There should only be 2 default table 0 flows"
```
Comment (ulaskozat) [inline lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py:589]: this should be 1 after ipfix changes, no?

Comment (koolzz) [reply lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py:589]: hm no, the ipfix flow is still there, just with higher pri

Comment (ulaskozat) [reply lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py:589]: ah, okay. for a moment I thought they were removed as well.

File: lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py (hunk 1 above)
Comment (pruthvihebbani) [reply lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py]: I took this piece of code from test_attach_detach_rar_tcp_data.py TC assuming that flows with priority 0 are the default rules which are there even when the TCs are not running

Comment (ulaskozat) [reply lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py]: @pruthvihebbani raised and merged PR #7013.

**Comment by themarwhal** [comment]
Removing backport as 1.5 doesn't have this function: `verify_flow_rules_deletion`

**Pull request merged**